plugin/src/test/groovy/com/novoda/test/TestAndroidProject.groovy

```diff
@@ -63,11 +63,6 @@ ${formatExtension(project)}
         .join('\n\t\t')
     }
 
-    @Override
```

Comment:
@mr-archano what was the reasons for this? Can we kill it? All tests are still passing...

Reply:
the reason for this was to temporary disable lint as the default configuration breaks the build on error. I would have expected the other integration tests to fail if these lines are removed, mmm